README.md

```diff
@@ -559,6 +584,8 @@ let person = [%obj {
   };
   age = 32
 }]
+
+let _ = Js.log (## (## person name) first)
```

Comment:
This isn't valid OCaml. Looks like you wrote it in Reason and converted it in the Try Reason playground which doesn't handle `##` properly. It should be `let _ = Js.log (person##name##first)`.

I'm not sure this line should be there at all though, since the example should illustrate object creation. not object access, and could be confusing.

Reply:
You are right. It should be removed. 

HAHA. You are a wise man. I am an newbie. I use `refmt --parse ml --print re --interface false` to convert Ocaml syntaxt to Reasonml.  I have to lookup Ocaml docs when I meet problems.

In fact, I am very confused. why create a new syntax , named Reasonml? Just for marketing?

Reply:
Mostly to make it more familiar and approachable to developers already familiar with JavaScript or other Algol-style languages, I think, which is a pretty sizable share of all developers.

But it also fixes a many inconsistencies and general weirdness with OCaml. I prefer Reason, even if it does have its issues too.